.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9]
```

Comment:
Should we include Python 3.7 and 3.8 here? Since we're continuing to support them.

Reply:
I added 3.8, 3.11, and 3.12...not sure why you aren't seeing them

Reply:
I see them now. Python 3.7 is still missing here though. I think we could just drop support for it at this point, but then we should make that change explicitely in `setup.py` and `pyproject.toml` as well.

Reply:
on it